src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+// debugger;
+const populationSpan = document.querySelectorAll('.population');
+
+const population = [...populationSpan].map(value =>
+  (
+    +[...value.textContent].join('').split(',').join('')
```

Comment:
Тут думаю зайве іде створення масиву щоб його одразу обєднати 😅
```suggestion
    +value.textContent.split(',').join('')
```
Так має бути той же результат, якщо помиляюся пиши..)

Reply:
мдя .... перемудрил что-то, спасибо 👍